test-harness/createMasterFiles.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const requests = require('./requests');
+
+const { makeRequest, constructMasterFileName } = require('./helpers');
+
+async function recordMasterFile({ path, method, headers, body }) {
+  const reqRes = await makeRequest({ path, method, headers, body });
+
+  try {
+    fs.writeFileSync(
+      `${__dirname}/gold-master-files/${constructMasterFileName({
+        path,
+        method,
+        headers,
+        body,
+      })}.json`,
+      JSON.stringify(reqRes, null, 2)
+    );
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+(async function() {
+  for (const request of requests) {
+    const { dynamic, ...req} = request;
+
+    await recordMasterFile(req);
```

Comment:
I think all these request can be run using `Promise.all`, decreasing the run time kind of significantly

```ts
await Promise.all(requests.map(recordMasterFile))
```

Reply:
thanks, great tip :+1: